docs/kyma/docs/032-gs-cluster-installation.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Cluster Kyma installation
+type: Getting Started
+---
+
+## Overview
+
+This Getting Started guide shows developers how to to quickly deploy Kyma on a cluster. Kyma installs on a cluster using a proprietary installer based on a Kubernetes operator. The document provides prerequisites, instructions on how to install Kyma on a cluster and verify the deployment, as well as the troubleshooting tips.
+
+## Prerequisites
+
+**NOTE:** The public IPs and DNS records for Istio Ingress and the remote environments gateway must exist prior to the Kyma installation. Also, the Kyma installer does not support clusters on AWS because AWS does not support static IP assignment during ELB creation.
+
+Before you start the installation, check that you have access to a running cluster with the minimum Kubernetes version 1.10.
```

Comment:
How about:

The cluster on which you install Kyma must run Kubernetes version 1.10 or higher. 

?

Reply:
Sure, why not.